screed/__main__.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2016, The Regents of the University of California.
+
+from __future__ import absolute_import, print_function
+
+import argparse
+import sys
+
+from . import createscreed
+from . import dump_to_fasta
+from . import dump_to_fastq
+
+
+class ScreedCommands(object):
+
+    def __init__(self):
+        parser = argparse.ArgumentParser(
+            description="",
+            usage='''screed <command> [<args>]
+
+Commands can be:
+
+    db <filename>                 "Creates a screed database."
+    dump_to_fasta <db> <output>   "Convert a screed database to a FASTA file"
+    dump_to_fasta <db> <output>   "Convert a screed database to a FASTQ file"
```

Comment:
I'd be happy to take care of this, but any comments on `dump_to_fastq` vs just `fastq`?